modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -876,7 +878,7 @@ class CV_EXPORTS_W VideoWriter
     @param fps Framerate of the created video stream.
     @param frameSize Size of the video frames.
     @param isColor If it is not zero, the encoder will expect and encode color frames, otherwise it
-    will work with grayscale frames (the flag is currently supported on Windows only).
+    will work with grayscale frames (the flag is ignored on MSMF and Intel MFX backends).
```

Comment:
> the flag is ignored

It is better to keep here common information only.
Backend-specific information should be moved on dedicated pages (there is huge amount of implementation details)

Reply:
Does `(the flag may be ignored by some backends. For more information refer to backend implementation)` sound better?

Reply:
I would prefer to keep this info in a single place (to avoid manual maintenance in different places).

See here:
https://github.com/opencv/opencv/blame/4.3.0/modules/videoio/doc/videoio_overview.markdown#L27

I believe the header of properties enumeration should be enough.

Reply:
Thank you, got it!